contracts/src/emergency/sequencer_uptime_feed.cairo

```diff
@@ -0,0 +1,305 @@
+impl Felt252TryIntoBool of TryInto::<felt252, bool> {
+    fn try_into(self: felt252) -> Option<bool> {
+        if self == 0 {
+            Option::Some(false);
+        }
+        if self == 1 {
+            Option::Some(true);
+        }
+        Option::None(())
+    }
+}
+
+impl BoolIntoFelt252 of Into::<bool, felt252> {
+    fn into(self: bool) -> felt252 {
+        if self {
+            return 1;
+        }
+        return 0;
```

Comment:
```suggestion
        if self {
            return 1;
        }
        0
```

Reply:
Do we actually need these traits now? I'd also consider opening a PR upstream

Reply:
yessir you're correct , we don't need them anymore 🥳